docs/howto/build-run.md

```diff
@@ -4,9 +4,10 @@ This guide describes how to build and run the app so you can develop it.
 
 ## Main steps
 
-(First, if using **macOS**: Upgrade to the latest version of the OS and then
-to the latest Xcode.  In particular, Xcode versions before 9.0 are known to
-definitely not work.)
+(First, if using **macOS**: Upgrade to the latest version of the OS and then to
+the latest Xcode.  In particular, Xcode versions before 9.0 are known to
+definitely not work. You'll also need the GNU coreutils package from Homebrew,
+installable via `brew install coreutils`.)
```

Comment:
I hesitate to say "you must use Homebrew", given how bad Homebrew is.

The error message in `tools/lib/ensure-coreutils.sh` says
```
This script requires GNU coreutils.

Install from your favorite package manager, or from:
  https://www.gnu.org/software/coreutils/
```

Perhaps "You'll also need GNU coreutils installed, either from your favorite package manager or [upstream](https://www.gnu.org/software/coreutils/)."?

Reply:
We do already have a reference to `brew install coreutils` in debugging.md, and the [React Native instructions we reference](https://facebook.github.io/react-native/docs/getting-started.html) also give direct `brew` commands.

I'm not sure that's a battle worth fighting. :confused:

Reply:
Lol, I'd forgotten these lines (which I wrote):

    (On macOS, your `date` command may not have this feature, because
    it's a version whose UI hasn't changed since the '80s.  You want
    GNU `date`.  `brew install coreutils` will install it, with the name
    `gdate`.)

That still feels to me quite different from "you need the thing from Homebrew". It says "you need this thing; one way to get it is this Homebrew command".

I'm not particularly interested in taking on the battle of fighting Homebrew-normativity in other projects. :stuck_out_tongue: But I'd like to avoid contributing to it ourselves, and I think we can pretty easily accomplish that.

(OTOH I might at some point make the effort to switch to something else on our own office Mac, at least. Might leave Homebrew around just to refer to how it works when helping a contributor debug their setup... but could eliminate actually relying on it.)

Reply:
> Perhaps "You'll also need GNU coreutils installed, either from your favorite package manager or [upstream](https://www.gnu.org/software/coreutils/)."?

Or a version that mentions `brew install` without saying Homebrew is mandatory could be

"You'll also need [GNU coreutils](https://www.gnu.org/software/coreutils/) installed, e.g. with `brew install coreutils`."